src/types.js

```diff
@@ -46,9 +46,10 @@ export type Dependency = {
   license: string,
   repository: Repository,
   // All of the above fields are straight from the dependency's package.json.
-  // The status field is separate, and used to show loading indicators while
-  // performing actions on the dependency.
+  // The status/location fields are separate, and used to show loading indicators while
+  // performing actions on the dependency, and whether or not it's a devDependency.
```

Comment:
I think now that there are two fields, the comment could be restructured a bit:

```js
// All of the above fields are straight from the dependency's package.json.
// The following two are derived values:
// `status` is used to show loading indicators while performing actions on the dependency
status: DependencyStatus,
// `location` refers to where the dependency lives in the host project
location: 'dependencies' | 'devDependencies',
```

(also I updated the type, since we know it's one of two values, not any string)